src/web/theme/modules/Tile/Tile.js

```diff
@@ -1,22 +1,28 @@
 import React from "react";
 import PropTypes from "prop-types";
-import ContentTile from "./ContentTile";
 import "./Tile.scss";
 
 const Tile = ({ title, children, style, contentStyle }) => {
   const TileClass = `tile${style ? " tile--" + style : ""}`;
+  const contentClass = `tile__content${
```

Comment:
I think it was the same in the previous PR so this is not important for this one, but next time try to remember to use proper BEM for your modifiers :slightly_smiling_face: